src/plays/inspirational-quotes/InspirationalQuotes.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import PlayHeader from "common/playlists/PlayHeader";
+import Card from "./Card";
+import Quotes from "./Quotes";
+import "./inspirational-quotes.css";
+
+export default function InspirationalQuotes(props: any) {
+  const [quote, setQuote] = useState(Quotes[2]);
```

Comment:
Can you add the `QutoteType` for the quote?
```suggestion
  const [quote, setQuote] = useState<QutoteType>(Quotes[2]);
```